glossary.md

```diff
@@ -0,0 +1,43 @@
+# Glossary of Terms
+
+OpenAttestation: The underlying technology/framework that we use to power verifiable credentials
+
+OpenCerts: The project that is owned by SSG and is concerned with educational credentials
+
+TradeTrust: The project that is owned by IMDA and is concerned with trade documents as well as transferable asset deeds
+
+### OpenAttestation Framework
+  
+  Envelope: Wrapper around a document that uses OpenAttestation framework to protect its integrity. _(Sometimes called Signed JSON file)_
+  
+  Proof: Part of the `Envelope` metadata, a set of Keccak256 hashes that leads to the Merkle root that is notarised (i.e on the Ethereum blockchain)
+  
+  Document: The information inside the .opencert or .tt file that excludes the `Envelope` metadata. _(Sometimes called Unsigned JSON)_
+  
+  Schema: Refers to a specific schema that the contained `Document` has to adhere to.
+  
+  Template: Instructions on how to display the data in the `Document`, could be a URL pointing to a `Renderer` or simply a path depending on OpenAttestation version
+  
+  Renderer: A HTTP(s) endpoint that takes a `Document` and returns some browser-renderable contents (i.e HTML/CSS/JS)
```

Comment:
```suggestion
  Renderer: A website to be embedded in the OpenAttestation platform. It should take a `Document` and render it in browser. A sample application is available in the `decentralized-renderer` repository.
```

Reply:
Wanted to highlight that is is actually rendered.

Reply:
technically the browser renders it, the url just takes the document and returns some code for the browser to render